ibm_concert/cert_renewal/README.md

```diff
@@ -0,0 +1,66 @@
+# Manage z/OS Certificates using IBM Concert, ServiceNow, and Ansible Automation Platform
+
+These sample playbooks when used with IBM Concert, ServiceNow, and Ansible Automation Platform
+can simplify the monitoring and resolution of RACF certificate issues.
+
+It is a good practice to review the playbook contents before executing them.
+It will help you understand the requirements in terms of space, location, names,
+authority, and the artifacts that will be created and cleaned up.
+
+## Playbook Requirements
+These playbooks use:
+
+    collections:
+      - name: ibm.ibm_zos_core
+        version: 1.10.0
+      - name: ansible.posix
+        version: 1.5.4
+      - name: ansible.utils
+        version: 4.1.0
+      - name: community.general
+        version: 9.4.0
+      - name: servicenow.itsm
+        version: 2.7.0
+
+## Playbooks
+[**send_cert_data.yml**](send_cert_data.yml) this playbook will run the z/OS Health Checker RACF Certificate Expiration report and pull the data into a CSV file and send it to an IBM Concert instance for visualization and management.
+
+[**renew_cert.yml**](renew_cert.yml) this playbook will renew a z/OS certificate using the certificate data sent in from a ServiceNow REST message using a job template on Ansible Automation Platform.
+
+## Role Summary
+- [**get_cert_detail**](roles/get_cert_detail/README.md) - Retrieve certificate details from RACF and build a CSV record
+- [**issue_operator_cmd**](roles/issue_tso_cmd/README.md) - Issue z/OS system command(s)
+- [**issue_tso_cmd**](roles/issue_tso_cmd/README.md) - Issue TSO command(s)
+- [**print_hc_buffer**](roles/print_hc_buffer/README.md) - Pull data from Health Checker
+- [**send-template**](roles/send-template/README.md) - send template to a zOS host
+
+## Set up job templates on Ansible Automation Platform
+These playbooks are designed to be used with Ansible Automation Platform (AAP) job templates. The information in the `host_vars` can be used to set up Inventory and Hosts on AAP.
+
+Review the required inputs to each playbooks to set up Surveys on AAP so that external callers can call the AAP REST API correctly.
+
+- Set up an AAP schedule to send certificate data to IBM Concert on a regular basis
+- Set up an AAP job template to renew a certificate data to IBM Concert on a regular basis
```

Comment:
is this supposed to say "Set up an AAP job template to renew a certificate on a regular basis" ?

Reply:
yes